pkg/agent/agent.go

```diff
@@ -103,6 +97,12 @@ func (k *Klocksmith) process(stop <-chan struct{}) error {
 		return err
 	}
 
+	// we are schedulable now.
+	glog.Info("Marking node as schedulable")
+	if err := k8sutil.Unschedulable(k.nc, k.node, false); err != nil {
```

Comment:
@sdemos ack, let's move this to its own ticket/PR. Perhaps there is aready one, I didn't check (my bad). Can you take care of that?